crates/matrix-sdk-crypto/src/types/cross_signing/self_signing.rs

```diff
@@ -33,6 +34,14 @@ impl SelfSigningPubkey {
         &self.0.usage
     }
 
+    /// Get the first available self signing key.
```

Comment:
This comment is a bit strange. The first available? It's just getting the ed25519 key and there is only one?

Reply:
Well, the line just below does explain that there is probably only one key.

Honestly I'm not a big fan of this API either, but it's just cargo-culted from the `MasterPubkey` implementation.